src/themes/default/modules/checkbox.variables

```diff
@@ -149,20 +149,20 @@
 /* Toggle */
 @toggleLaneWidth: 3.5rem;
 @toggleHandleSize: 1.5rem;
-@toggleTransitionDuration: 0.2s;
+@toggleTransitionDuration: 0.3s;
```

Comment:
Why increased ?

Reply:
`0.3s` from `toggleTransitionDuration` 
https://github.com/fomantic/Fomantic-UI/blob/863d7123323f642238daaa5c92f005073fb9e204/src/themes/default/modules/checkbox.variables#L115

Reply:
Yup, that's why :)